lib/rubocop/cop/i18n/gettext/decorate_string.rb

```diff
@@ -4,25 +4,26 @@ module RuboCop
   module Cop
     module I18n
       module GetText
-        # This cop checks for butt or ass,
-        # which is redundant.
-        #
+        # This cop checks for sentence-like strings that are undecorated
         # @example
         #
         #   # bad
         #
-        #   "result is #{something.to_s}"
+        #   "The result is success."
         #
         # @example
         #
         #   # good
         #
-        #   "result is #{something}"
+        #   _("The result is success.")
         class DecorateString < Cop
+          SUPPORTED_DECORATORS = ['_', 'n_', 'N_']
```

Comment:
This constant should probably be abstracted out somewhere, seeing as it is being re-used in all the cops.

Reply:
Probably not in this PR, I'm just saying! 😆